src/main/java/kr/codesqaud/cafe/controller/LoginController.java

```diff
@@ -0,0 +1,29 @@
+package kr.codesqaud.cafe.controller;
+
+import kr.codesqaud.cafe.domain.Member;
+import kr.codesqaud.cafe.service.MemberService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.servlet.http.HttpSession;
+
+@Controller
+public class LoginController {
+    private MemberService memberService;
+
+    @Autowired
+    public LoginController(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    @PostMapping("/login")
+    public String login(Member member, HttpSession httpSession) {
+        Member loginMember = memberService.findOneMemberByEmail(member.getEmail()).orElse(null);
+        if (loginMember != null && memberService.checkMember(loginMember, member)) {
+            httpSession.setAttribute("sessionedUser", loginMember);
+            return "redirect:/";
+        }
```

Comment:
orElse(null) 은 꼭 필요한 경우가 아니면 지양해 주세요.
```suggestion
        Member loginMember = memberService.findOneMemberByEmail(member.getEmail());
        if (loginMember.isPresent()) { //...
```

Reply:
orElseThrow()로 수정했습니다.